x-pack/legacy/plugins/apm/server/lib/traces/get_trace_items.ts

```diff
@@ -44,14 +44,15 @@ export async function getTraceItems(traceId: string, setup: Setup) {
         { _score: { order: 'asc' } },
         { [TRANSACTION_DURATION]: { order: 'desc' } },
         { [SPAN_DURATION]: { order: 'desc' } }
-      ]
+      ],
+      track_total_hits: true
```

Comment:
I'm wondering if we should set `track_total_hits=10` as default in `setup_request` since we only need the exact count here, and elsewhere we just need to know if there are any results.

Reply:
(out of scope for this PR but worth looking into in the future)